src/System.Net.Http/src/System/Net/Http/Unix/CurlHandler.cs

```diff
@@ -15,11 +17,13 @@
 
 namespace System.Net.Http
 {
-    internal class CurlHandler : HttpMessageHandler
+    internal partial class CurlHandler : HttpMessageHandler
     {
         #region Constants
 
+        private const string UriSchemeHttp = "http";
         private const string UriSchemeHttps = "https";
+        private static bool s_CurlInitialized = InitializeCurl();
```

Comment:
See comment below, but if you do end up keeping this, it should be readonly, and the name should be ```s_curlInitialized``` rather than ```s_CurlInitialized```.